<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wichat</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,2">
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="6,1,2,1">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QTabWidget" name="tabSession">
         <widget class="QWidget" name="tabDefault">
          <attribute name="title">
           <string>Tab 1</string>
          </attribute>
          <widget class="QTextBrowser" name="textBrowser">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>411</width>
             <height>171</height>
            </rect>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frameTextCtrl">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="buttonFont">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>41</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QPushButton" name="buttonEmotion">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>0</y>
            <width>41</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit"/>
       </item>
       <item>
        <widget class="QFrame" name="frameMsgCtrl">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="buttonSend_2">
          <property name="geometry">
           <rect>
            <x>290</x>
            <y>0</y>
            <width>111</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
         <widget class="QPushButton" name="buttonSendOpt">
          <property name="geometry">
           <rect>
            <x>401</x>
            <y>0</y>
            <width>20</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>â–¼</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListView" name="listFriend"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
